JoyboyCommunity/src/assets/icons.tsx

```diff
@@ -189,3 +189,64 @@ export const SendIcon: React.FC<SvgProps> = (props) => {
     </Svg>
   );
 };
+
+export const RepostIcon: React.FC<SvgProps> = (props) => {
+  return (
+    <Svg width="32" height="32" viewBox="0 0 21 21" {...props}>
+      <G
+        fill="none"
+        fill-rule="evenodd"
+        stroke="currentColor"
+        stroke-linecap="round"
```

Comment:
Svg attributes should be camelCased, for example `stroke-linecap` should be `strokeLinecap`